### ISSUE TITLE
Crash once seed is analyzed in analyzer.

### ISSUE THREAD

**Issue by Slither88** [InfinityRaider/AgriCraft#112]
Hi

I get a crash once the analyser has finished analysing the seed. Once I get back into the game the analysis is complete but there is no stat change. This is for crops from Harvestcraft and Natura so far. I was trying to breed super crops in a personal pack.

Crash Report: http://pastebin.com/8uV7TCWz

**Comment by InfinityRaider** [comment]
This is on 1.2.2, Update to 1.3.0 and see if it persists please.

**Comment by Slither88** [comment]
Will try that and feedback.

**Comment by Slither88** [comment]
I think I am getting a different crash. http://pastebin.com/P2bnzFBm

**Comment by Slither88** [comment]
Wait is just a villager ID clash it seems. I am testing again.

**Comment by Slither88** [comment]
I still get the same crash. Even with the latest version of agricraft. http://pastebin.com/7qU5WuA2

**Comment by InfinityRaider** [comment]
Okay, but at least now the line numbers add up and I can debug decently, thanks.

**Comment by InfinityRaider** [comment]
God dammit someone registered an ItemStack without an item to the oredict -.-, do I really have to add a null check for that.
I really would like to know what mod is responsible for that.

**Comment by Slither88** [comment]
You and me both. Thanks for the replies and good luck.

**Comment by Slither88** [comment]
I think I can help with finding out which mod it is. I have two packs that I use and in the one it crashes the client but not in the other. I will see if I can find the culprit for you if that will help.

**Comment by Slither88** [comment]
Hi there also seems to be a crash when I try to page through the journal.

http://pastebin.com/AZqLp4Z7

Don't know if this is a separate issue. If it is I will create a new ticket.

**Comment by Slither88** [comment]
Strangest thing is neither of the crashes occur when I load a new map. Just an old one that I have been playing on for a while now. I have added and removed a few mods will just test with all the mods I had when the crash firs occurred and then I think it might just be something that one of the removed mods changed in the world somehow. Don't know if this helps at all though.

**Comment by Slither88** [comment]
Ok I think I kind of understand why it crashes. I had magical crops installed and removed it but I analysed a seed I think from magical crops before removing it. The latest version of agricraft does not seem to like magical crops very much. I reported it #115. So now that I have re-added magical crops and reverted to the older (1.2.2) version of agricraft no crashes. Hope this helps. For now I will use the older version of agricraft.

**Comment by InfinityRaider** [comment]
Okay, for making it easyer for me, can you list all the crashes you are enocuntering in 1.3 so I can add in null checks where necessary?

**Comment by Slither88** [comment]
So far I get crashes when starting up the world with agricraft 1.3.0 b407 and magical crops present but not with agricraft 1.2.2 and Magical crops present. I will update both and see if it still crashes, but I see you have already fixed this one.
